engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ConstraintValidatorContextImpl.java

```diff
@@ -218,8 +227,8 @@ protected ConstraintViolationBuilderImpl(String template, PathImpl path) {
 		}
 
 		@Override
-		public HibernateConstraintViolationBuilder enableExpressionLanguage() {
-			expressionLanguageEnabled = true;
+		public HibernateConstraintViolationBuilder enableExpressionLanguage(ExpressionLanguageFeatureLevel expressionLanguageFeatureLevel) {
+			this.expressionLanguageFeatureLevel = ExpressionLanguageFeatureLevel.interpretDefaultForCustomViolations( expressionLanguageFeatureLevel );
```

Comment:
When you pass `DEFAULT` to this method, shouldn't it set the feature level to `defaultCustomViolationExpressionLanguageFeatureLevel`?

Reply:
Hmmm, I think I prefer it being a static default. I'll see if people complain about it.